recipes/ocp/meta.yaml

```
@@ -0,0 +1,70 @@
+{% set name = "ocp" %}
+{% set version = "7.5.3" %}
```

It feels like the version number should be this:
```suggestion
{% set version = "7.5.3.0" %}
```
Is there a reason for being different?

---

Is 7.5.3.0 a valid version number for a package on conda-forge?

---

They follow PEP-440 which has no limit on the number of segments.

---

+1

---

The version is now 7.5.3.0